<ui version="4.0" >
 <class>InputTemplateEditor</class>
 <widget class="QDialog" name="InputTemplateEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Input Template Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="m_manufacturerLabel" >
         <property name="text" >
          <string>Manufacturer</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="m_manufacturerEdit" >
         <property name="toolTip" >
          <string>The name of the company that made the device</string>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="m_modelLabel" >
         <property name="text" >
          <string>Model</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="m_modelEdit" >
         <property name="toolTip" >
          <string>The device's model name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Channels</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item rowspan="5" row="0" column="0" >
        <widget class="QTreeWidget" name="m_tree" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick" >
          <bool>false</bool>
         </property>
         <column>
          <property name="text" >
           <string>Channel</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Type</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QToolButton" name="m_addButton" >
         <property name="toolTip" >
          <string>Add a new channel description</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="iconSize" >
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QToolButton" name="m_removeButton" >
         <property name="toolTip" >
          <string>Remove the selected channels</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="iconSize" >
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QToolButton" name="m_editButton" >
         <property name="toolTip" >
          <string>Edit the selected channel</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="iconSize" >
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QToolButton" name="m_wizardButton" >
         <property name="toolTip" >
          <string>Attempt to automatically detect available channels from an input device</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="iconSize" >
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>22</width>
           <height>131</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InputTemplateEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InputTemplateEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
